### ISSUE TITLE
IE11

### ISSUE THREAD

**Issue by crisward** [matthewp/haunted#34]
A bit of a longshot, but has anyone got this working in IE11, perhaps with babel transpiling? Got it working in Edge, but IE11 seems to trip up on the template literals. I'm using it with lit-html as per examples.

**Comment by matthewp** [comment]
Let me know what you find out. If you're only using virtual components it should work in any browser lit-html works in.

**Comment by Hotell** [comment]
AFAIR lit-html still doesn't work in IE11

**Comment by crisward** [comment]
I'll give it a go with hyperHTML, they claim IE9>, though I'm not sure how when using template literals. Intrigued to find out.

**Comment by crisward** [comment]
Tried with both lit and hyper, with polyfills and babel transpiling. Still have IE11 issues.

Lit does claim ie11 support - https://lit-html.polymer-project.org/, so in theory it should work. 

Would love to see a working sample if anyone else can get it working in IE11. With all the transpiling and polyfills the JS bundle leaps up in size, and no doubt performance probably plummets, but having proof it's possible would be cool (at least until IE dies...).

**Comment by askbeka** [comment]
Can you share how you transpile and build configs. Babel and webpack config
if you use webpack or other tools config

**Comment by crisward** [comment]
Added it here.
https://github.com/crisward/haunted-ie11

**Comment by Hotell** [comment]
awesome news ! can you maybe send a PR with tweaked docs ? 👌

**Comment by crisward** [comment]
Also found an issue with useEffect as IE11 doesn't have Symbols. I've hacked a fix, I'll do a quick pull request, but I can understand if you prefer to keep them.

**Comment by matthewp** [comment]
Yeah, if someone wouldn't mind updating the README and putting a section about IE11 support (that the polyfill above is needed) that would be fantastic. Probably in the [Use section](https://github.com/matthewp/haunted#use)

**Comment by matthewp** [comment]
Fixed by #36. Thanks @crisward !

**Comment by lsimone** [comment]
@crisward I tried to run your repository: it works just if you don't add the option `{ useShadowDOM: false }`

```
customElements.define('my-counter', component(Counter, { useShadowDOM: false }));
```
results in an error in IE11.
Do anybody know where it could come from?

![image](https://user-images.githubusercontent.com/3266650/73016686-a23bf200-3e1e-11ea-9701-880d78fe4a75.png)
